### ISSUE TITLE
COLABFOLD_BATCH not running on cpus after image update

### ISSUE THREAD

**Issue by JoseEspinosa** [nf-core/proteinfold#214]
After updating colabfold image to enable running it on cuda 12 versions on #195, it now gives the following error when running on cpus:

```
Traceback (most recent call last):
    File "/localcolabfold/colabfold-conda/bin/colabfold_batch", line 8, in
      sys.exit(main())
    File "//localcolabfold/colabfold-conda/lib/python3.10/site-packages/colabfold/batch.py", line 2046, in main
      run(
    File "//localcolabfold/colabfold-conda/lib/python3.10/site-packages/colabfold/batch.py", line 1286, in run
      if jax.local_devices()[0].platform == 'cpu':
    File "//localcolabfold/colabfold-conda/lib/python3.10/site-packages/jax/_src/xla_bridge.py", line 1135, in local_devices
      process_index = get_backend(backend).process_index()
    File "//localcolabfold/colabfold-conda/lib/python3.10/site-packages/jax/_src/xla_bridge.py", line 1011, in get_backend
      return _get_backend_uncached(platform)
    File "//localcolabfold/colabfold-conda/lib/python3.10/site-packages/jax/_src/xla_bridge.py", line 990, in _get_backend_uncached
      bs = backends()
    File "//localcolabfold/colabfold-conda/lib/python3.10/site-packages/jax/_src/xla_bridge.py", line 890, in backends
      raise RuntimeError(err_msg)
  RuntimeError: Unable to initialize backend 'cuda': FAILED_PRECONDITION: No visible GPU devices. (you may need to uninstall the failing plugin package, or set JAX_PLATFORMS=cpu to skip this backend.)
```

**Comment by JoseEspinosa** [comment]
Fixed by #217

**Comment by JoseEspinosa** [comment]
For cross-reference "Seems that the problem has been already reported [here](https://github.com/YoshitakaMo/localcolabfold/issues/238#issuecomment-2203777538)"
